analyses/cell-type-wilms-tumor-06/scripts/prepare-fetal-references.R

```diff
@@ -0,0 +1,215 @@
+#!/usr/bin/env Rscript
+
+# This script prepares references uses for label transfer, which will be performed with
+# code adapted from Azimuth. Therefore, references should be in a format expected by Azimuth, and
+# then they need to be further prepared for use with our adapted code.
+#
+# Two references are prepared:
+# - A fetal kidney atlas (Stewart) obtained from CELLxGENE. This is saved to <output_dir>/stewart_formatted_ref.rds
+# - A fetal organ reference (Cao) obtained from Azimuth. This is saved to <output_dir>/cao_formatted_ref.rds
+#
+# The script also downloads and saves an ID mapping file from Azimuth which will be used to convert ensembl IDs to
+#  gene names during label transfer. This is saved to <output_dir>/homologs.rds
+#
+# USAGE:
+# Rscript prepare-fetal-references.R \
+#   --kidney_url https://datasets.cellxgene.cziscience.com/40ebb8e4-1a25-4a33-b8ff-02d1156e4e9b.rds \
```

Comment:
For a few reasons (transparency, easier translation to a workflow, etc.), I think it would be better to separate downloads out of this script and instead have the downloads done in a wrapper script (that could also check if the downloads have already happened).